docs/user/gcp/install_xpn.md

```diff
@@ -0,0 +1,167 @@
+## Configure cloud firewall rules
+
+Because firewall rules are attached to the network, they must exist in the Host
+Project. As a result, additional permissions would be necessary to manage these
+rules. Choose one of the options below based on if you wish to enable these
+permissions:
+
+### Option A (Default): Automatic firewall rules (requires permissions)
+
+The default behavior is to automatically create and manage firewall rules. This
+option requires the installer service account to have the following permissions
+in the Host Project:
+
+- Compute Network Admin
+- Compute Security Admin
+
+### Option B: Manually created, pre-exsiting firewall rules
+
+When these permissions are not desired, it is possible to create firwall rules
+in advance and configure the installer to skip their creation. It is important
+to ensure the rules are sufficient to enable all cluster communications.
+Creation of firewall rules can be disabled by adding the following to the
+install-config.
+
+```
+platform:
+  gcp:
+    createFirewallRules: Disabled
+```
+
+The pre-existing rules can be cofigured specifically for the cluster instances
+by using network tags in combination with the following tags paremeters in the
+install-config:
+
+```
+compute:
+- name: worker
+  platform:
+    gcp:
+      tags:
+      - mycomputetag
+controlPlane:
+  platform:
+    gcp:
+      tags:
+      - mycontrolplanetag
+platform:
+  gcp:
+    defaultMachinePlatform:
+      tags:
+      - mydefaulttag
+```
+
+## Configure Public DNS Zone
+
+The public DNS zone can exist in either the Service Project or any other
+project. Because the public zone is authorative, it is common for it to exist
+in the Host Project.  As a result, when the public DNS Zone is not in the
+Service Project, additional premissions would be necessary to manage these
+resources. Choose one of the options below based on where your public DNS Zone
+exists, and if you wish to enable these permissions:
+
+### Option A (Default): Automatic in Service Project
+
+The default behavior is to find the DNS Zone in the Service Project and use it
+to create and manage DNS records. No special permissions are required.
+
+### Option B: Automatic in Host Project (Requires Permissions)
+
+Use the following install-config parameters to configure the cluster to use a
+DNS Zone in the Host Project.
+
+```
+platform:
+  gcp:
+    publicDNSZone:
+      id: my-public-zone-id
+      project: my-project-id
+
+```
+
+This option requires the installer service account to have the following
+permissions in the specified project:
+
+- DNS Administrator
+
+### Option C: Manual/External DNS management.
+
+This option is not currently supported.
+
+## Configure private DNS Zone
+
+The private DNS zone has the same considerations as the public zone. However,
+it is highly recommended to leave the private DNS zone in the service project
+as configured by default. This should not generally be a problem as these zones
+map 1 to 1 with clusters.
```

Comment:
nit: Typically it is practical to write out the full number when it is less than ten. 

"map one to one with clusters."